general/device.hpp

```diff
@@ -179,7 +189,13 @@ class Device
        * Multiple backends can be configured at the same time.
```

Comment:
Update the backend priority list in the documentation of `Device:: Configure`?

Reply:
Done.